packages/shared/lib/seeders/connection.seeder.ts

```diff
@@ -24,14 +24,23 @@ export const createConnectionSeeds = async (env: DBEnvironment): Promise<number[
     return connectionIds;
 };
 
-export const createConnectionSeed = async (env: DBEnvironment, provider: string, endUser?: EndUser): Promise<NangoConnection> => {
-    const name = Math.random().toString(36).substring(7);
+export const createConnectionSeed = async (
+    env: DBEnvironment,
+    provider: string,
+    endUser?: EndUser,
+    rest?: {
+        connectionId?: string;
+        rawCredentials?: AuthCredentials;
+        connectionConfig?: any;
+    }
+): Promise<NangoConnection> => {
+    const name = rest?.connectionId ? rest.connectionId : Math.random().toString(36).substring(7);
```

Comment:
🍖 make connectionId configurable through `rest`

Reply:
not a huge fan of this pattern. I am ok going with it because this change is time sensitive as long as we follow up immediately with a PR to use a object as parameter

Reply:
Its not a pattern its just like that for historical reason, changing to object will update a lot of tests so it was easier like this (we cant fix everything all the time unfortunately)
Happy to see it changed to object 👌🏻